docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md

```diff
@@ -37,6 +37,15 @@ When creating or modifying a Google Apps Audit Source, you will be required to a
 Authentication must be with a new G Suite Apps Audit Source, we do not support re-authenticating existing sources.
 :::
 
+### Manage Third party app access
+In order to provide Sumo logic access to google services, please follow the manage third-party app access steps as follows. See Google's [Manage Third-party App Access](https://support.google.com/a/answer/7281227#zippy=%2Cstep-manage-third-party-app-access-to-google-services-add-apps) documentation for more details. 
+1. In the Admin console, go to **Menu** > **Security** > **Access and data control** > **API controls**.
+1. In App access control, click **Manage Third-Party App Access**.
+1. Click **Add app** and search for Sumo Logic o-auth client.
+1. Select Sumo Logic o-auth client and click **Change access**. 
```

Comment:
@kparekh010 , can you confirm the exact wording (and capitalization) in the UI? Is it **Sumo Logic O-Auth client**?

Reply:
![gsuite-sumo-third-party-app](https://user-images.githubusercontent.com/98909252/221713997-d14bf76c-8c70-4910-a956-1f5e2c64095b.png)
Hi Kim, You can see the sumo logic web app in this image, this is the one customers need to add and give access to. I received this image from a current prospect.